docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,115 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
 If it's not the case, please refer to the following section(#)
 
+
+### Service Binding Resource
+
+```yaml
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-request 
+
+spec:
+
+  application:
+    name: nodejs-app
+    group: apps
+    version: v1
+    resource: deployments
+
+  services:
+  - group: postgresql.dev
+    version: v1alpha1
+    kind: Database
+    name: db-demo
+```
+
+There are 2 parts in the request:
+- application
+- services
+
+### Application
+
+An application can be referenced either by name or by labels.
+
+#### Reference by Name
+
+```
+  application:
+    name: nodejs-app
+    group: apps
+    version: v1
+    resource: deployments
+```
+
+#### Reference by Label Selector
+
+```
+  application:
+    matchLabels:
+      app: frontend
+    group: apps
+    version: v1
+    resource: deployments
+```
+
+### Service
+
+One or more services can be bound with application.
```

Comment:
No, we have removed that capability.
In one Service Binding resource, it's possible to declare only one app and one service.
Please @pedjak correct me if I'm wrong

Reply:
We still have the capability to connect multiple services -> https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToMultipleServices.feature#L84

However, there is a github issue to remove this capability to align with the spec -> https://github.com/redhat-developer/service-binding-operator/issues/898